src/test/java/org/fundacionjala/coding/dennis/VigenereTest.java

```diff
@@ -0,0 +1,91 @@
+package org.fundacionjala.coding.dennis;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class VigenereTest {
+
+    private Vigenere test;
+
+    /**
+     *
+     */
+    @Before
+    public void initial() {
+        test = new Vigenere();
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testVigenNormal() {
+        String exp = "TXYP XMUNHCRC";
+        String res = test.crypto("LIMON", "HOLA JALASOFT");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "LIMON" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)